Eatery/Onboarding/Controllers/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  OnboardingPageViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import CHIPageControl
+import UIKit
+
+class OnboardingPageViewController: UIPageViewController {
+
+    private var pages = [OnboardingViewController]()
+    private let pageControl = CHIPageControlJaloro()
+
+    private let models = [
+        OnboardingModel(title: "Menus", subtitle: "See what’s being served at any campus eatery.", image: UIImage(named: "menuIcon")!),
```

Comment:
Here are two possibilities: 
1. Not unwrap the image: `UIImage?` is optional in `OnboardingModel`
2. Remove `OnboardingModel` entirely